### PR TITLE
☂️ BaseViewManagerTest.java => BaseViewManagerTest.kt

### PR DESCRIPTION
**Pull request by retyui**
## Summary:

Issue: https://github.com/facebook/react-native/issues/38825#issuecomment-1669202200

## Changelog:

[GENERAL] [CHANGED] - `BaseViewManagerTest.java` => `BaseViewManagerTest.kt`

## Test Plan:

```shell
./gradlew :packages:react-native:ReactAndroid:test
```

### CODE REVIEW

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/BaseViewManagerTest.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import java.util.Locale
+
+@PrepareForTest(Arguments::class)
+@RunWith(RobolectricTestRunner::class)
+@PowerMockIgnore("org.mockito.*", "org.robolectric.*", "androidx.*", "android.*")
+class BaseViewManagerTest {
+    private lateinit var mViewManager: BaseViewManager<ReactViewGroup, *>
+    private lateinit var mView: ReactViewGroup
```
Comment (rshest) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/BaseViewManagerTest.kt:33]: Please don't use Hungarian notation in Kotlin (i.e. Kotlin-y names would be `viewManager` and `view`).

**Review by rshest — CHANGES_REQUESTED** [review]

**Comment by rshest** [comment]
@retyui - thanks for working on this! Looks good overall, I just have one small nitpick regarding hungarian notation :)

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/BaseViewManagerTest.kt (hunk 1 above)
Comment (retyui) [reply packages/react-native/ReactAndroid/src/test/java/com/facebook/react/uimanager/BaseViewManagerTest.kt:33]: done

**Review by rshest — APPROVED** [review]

**Review by cortinico — APPROVED** [review]
LGTM

**Pull request closed without merging**